styles.go

```diff
@@ -2,11 +2,22 @@
 
 package promptui
 
-// Icons used for displaying prompts or status
+// These are the default icons used bu prompt-ui for select and prompts. They can either be overridden directly
+// from these variable or customized through the use of custom templates
 var (
+	// IconInitial is the icon used when starting in prompt mode and the icon next to the label when
+	// starting in select mode.
 	IconInitial = Styler(FGBlue)("?")
+
+	// IconGood is the icon used when a good answer is entered in prompt mode.
 	IconGood    = Styler(FGGreen)("✔")
+
+	// IconWarn is the icon used when a good, but potentially invalid answer is entered in prompt mode.
 	IconWarn    = Styler(FGYellow)("⚠")
+
+	// IconBad is the icon used when a bad answer is entered in prompt mode.
 	IconBad     = Styler(FGRed)("✗")
+
+	// IconBad is the icon used to identify the currently selected item in select mode.
 	IconSelect  = Styler(FGBold)("▸")
 )
```

Comment:
The names of these variables describe their use. I don't think additional comments are needed, but block level comments are great.

Reply:
I've made a type there, it's `iconSelect`, not `iconBad`.